lib/token_store.rb

```diff
@@ -38,8 +38,8 @@ def self.acquire(namespace, token_ttl)
           store.extend KeyValueHelpers
         end
       when "cache"
-        require 'active_support/cache/dalli_store'
-        ActiveSupport::Cache::DalliStore.new(MiqMemcached.server_address, options).tap do |store|
+        require 'active_support/cache/mem_cache_store'
```

Comment:
Interesting, it worked for me without the require. https://github.com/ManageIQ/manageiq/pull/21280/files#diff-93849d9b10bde9c5c0572f0697518484f24019280d73640c2b6b9feecea6978cL41-R41

Reply:
Probably better to have it though

Reply:
Oh, you are probably right that it isn't required since Rails probably includes `activesupport/all`, but I was just swapping out what was there previously.

Good catch though.  Thanks!